webpack.renderers.js

```diff
@@ -19,16 +23,13 @@ module.exports = (env) => {
         'inactivity-preload': './src/inactivity/preload.ts',
     }
     const basePlugins = [
-        new HtmlWebpackPlugin({
-            template: path.resolve(__dirname, 'static/pages/app.html'),
-            filename: 'app.html',
-            chunks: ['app'],
```

Comment:
Did you try to open the wexond application after deleting this ? It should not work anymore because this HtmlWebpackPlugin is injecting a chunk here that contains the entire application.